dev/com.ibm.ws.webcontainer.servlet.4.0/src/com/ibm/ws/webcontainer40/srt/ISRTServletRequest40.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.htmln *
+ * Contributors:
+ *      IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.webcontainer40.srt;
+
+import javax.servlet.http.HttpServletMapping;
+
+import com.ibm.ws.webcontainer40.osgi.webapp.WebAppDispatcherContext40;
+
+public interface ISRTServletRequest40 {
```

Comment:
Should this interface extend ISRTServletRequest so you don't have to cast to both if you need methods on each of them?